### PR TITLE
Add Jeremy Fiel as an ambassador

### PR DESCRIPTION
**Pull request by jeremyfiel**
Please let me know if I need anything further.

thanks 
Jeremy

### CODE REVIEW

**Review by Julian — APPROVED** [review]
There are changes here to other parts of the file other than the new section / the diff is large, which likely is the reason there's a merge conflict.

After backing those out though this has my support personally.

**Review by benjagm — APPROVED** [review]
Huge thanks for your interest in becoming an Ambassador. 
Everything looks great to me.

**Comment by benjagm** [comment]
> there's a merge conflict.

I just fixed it to make things easier.

File: programs/ambassadors/ambassadors.json [hunk 1]
```diff
@@ -279,5 +279,63 @@
...
+        },
+        {
+          "type": "other",
+          "title": "JSON Schema Stack Overflow Community Member",
+          "date": {
+            "year": 2021,
+            "month": "November"
+          },
+          "link": "https://stackoverflow.com/questions/tagged/json-schema"
+        },
```
Comment (gregsdennis) [inline programs/ambassadors/ambassadors.json:329]: I wonder if we need a "community presence" field for these.  This seems common enough to warrant its own field.

**Comment by jeremyfiel** [comment]
> There are changes here to other parts of the file other than the new section / the diff is large, which likely is the reason there's a merge conflict.
> 
> After backing those out though this has my support personally.

Some of the prior entries violated the enums and the formatting was incorrect

**Review by mwadams — APPROVED** [review]

File: programs/ambassadors/ambassadors.json (hunk 1 above)
Comment (benjagm) [reply programs/ambassadors/ambassadors.json:329]: > I wonder if we need a "community presence" field for these. This seems common enough to warrant its own field.

I just created a PR to add a new value in the enum referring to Community contributions:

https://github.com/json-schema-org/community/pull/791

**Review by jviotti — APPROVED** [review]

**Pull request merged**